src/contentful/types.ts

```diff
@@ -0,0 +1,105 @@
+export type NebenanLocales = 'de-DE' | 'de' | 'en-US' | 'en' | 'fr-FR' | 'fr' | 'es-ES' | 'es' | 'it-IT' | 'it';
+
+export type ContentfulSpaceDefaults = {
+  id: string;
+  token: string;
+  preview_token: string;
+};
+
+export type ContentfulSpace<T extends string | undefined = undefined> = T extends string
```

Comment:
once space has many different content type configurations. should we really define this is a generic here?

Reply:
maybe can pass in an array of content types? that would be crazy 😀

Reply:
Didn't really got this one :)

Reply:
```
      content_type_story: 'homeStory',
      content_type_promotion: 'homepageTeaser',
      content_type_stage: 'homepageSatge',
      content_type_press: 'pressLogos',
      content_type_team_stage: 'teamStage',
      content_type_team_member: 'teamMembers',
      content_type_partner: 'partnerLogoBar',
      content_type_city_story: 'cityOverviewStage',
      content_type_city_stage: 'city_stage',
      content_type_about_stage: 'aboutProductStage',
      content_type_features: 'aboutProductTabs',
      content_type_engage_steps: 'aboutProductSteps',
      content_type_rules_header: 'about_golden_rules',
      content_type_rules: 'about_rule',
      content_type_home_teaser: 'homepage_promotion_teaser',
      content_type_featured_banner: 'trust_banner',
      content_type_press_article: 'press_articles',
      content_type_vacancies: 'job',
      content_type_promo_video: 'employerBrandingVideo',
      content_type_legal: 'legalPages',
      content_type_presskit: 'presskit',
      content_type_donation_slides: 'donationSlides',
      content_type_donation_amounts: 'donation_amount',
      content_type_donation_page: 'donation_page',
      content_type_mitrust_slides: 'mitrust_slides',
      content_type_yes_com_slides: 'yes_com_slides',
      content_type_corona_partners: 'corona_partners',
```

This is how a config can look like. Your type supports passing **one** content type, which might not make sense since you can request multiple.